bundles/af-core/src/test/resources/form/password/exporter-password.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
You’ve made the same mistake I highlighted in the previous review—these files are not even in used in the test (while it should have being)

Why did you request a re-review when the issue from my earlier comment has still not been addressed?